_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,351 @@
+
+- **Multi-path payments:** the current normal way to make a payment over
+  LN is using a single path.  Alice pays Charlie through her channel to
+  Bob and Bob's channel to Charlie.  This works well for small payments
+  where each participant has enough capacity
+  to support the payment.  But if we use this mechanism when Alice has
+  10 open channels each containing a maximum of 10% of her total hot
+  wallet balance, Alice can only spend at most 10% of her funds at a
+  time.
+
+    Multipath payments provide a solution to this problem: if Alice
+    wants to send 15% of her funds, she can send 7.5% to Charlie through
+    her channel with Bob and 7.5% through her channel with Dan (who also
+    has a channel with Charlie).  Although the partial payments are
+    routed through separate paths, they can each commit to the same
+    hash Alice would've used to send a single-path payment.  If Charlie
+    receives multiple payments within a reasonable time period that
+    equal or exceed the expected amount, he can guarantee that he'll
+    receive all of them by simply revealing the single preimage used by
+    all of the hashes.  This reuses the same proven security mechanism
+    currently used for single-path payments and so doesn't introduce any
+    new security assumptions.  The same mechanism also works if some
+    other party along the path with sufficient channel capacity
+    merges together the partial payments and forwards a single
+    payment along the remainder of the path to Charlie.
+
+    For more information, see the following Lightning-Dev threads which
+    often call this feature Atomic Multi-path Payments (AMP): [an early
+    proposal with separate hashes/preimages][roasbeef amp], [something
+    like the currently favored proposal][zmn base amp], [a possibly too
+    optimistic proposal][pickhardt local amp].
+
+- **Dual-funded channels:** a nice feature of the current implementation
+  is that only one party to the channel needs to initially commit any
+  funds to it.  For example, Alice opens a channel to Bob with 0.1 BTC
+  of her money and none of Bob's money.  This makes it very easy for
+  users to accept new incoming channels, but it also means that channels can only be
+  used in one direction initially---Alice can pay Bob or route payments
+  through Bob, but Alice can't receive payments from Bob or from any
+  routing path including Bob until Alice has sent Bob some money.  This
+  creates a bootstrapping problem: if Alice wants to receive payments
+  via LN, she has to get people to open new channels to her node---which
+  requires they pay onchain transaction fees and wait for onchain
+  confirmations that can take hours.
+
+    A proposed solution to this problem is to allow dual-funded
+    channels.  Alice agrees to put 0.1 BTC into a channel with Bob if
+    Bob agrees to open the channel with 0.1 BTC of his own funds.  This
+    can cost Bob money---namely onchain transaction fees and lost
```

Comment:
nit: is 'lost opportunity cost' right? Should it be 'onchain transaction fees and the opportunity cost'